scripts/restore_epstage_data.py

```diff
@@ -0,0 +1,220 @@
+from datetime import datetime
+
+import pickle
```

Comment:
why pickle, and not just dump to json?

Reply:
Mainly because I thought of pickle first :sweat_smile: Are there any advantages going with json? I know the files will be human readable - I think viewing the deserialised dict in the terminal is going to be the preferred option, still.